src/pages/index.js

```diff
@@ -2,16 +2,23 @@
 import FormValidator from '../components/FormValidator.js';
 import Card from '../components/Card.js';
 import Section from '../components/Section.js';
+import Popup from '../components/Popup.js';
 import PopupWithForm from '../components/PopupWithForm.js';
```

Comment:
Please, delete `this._popupForm.reset();` from line 33 in `PopupWithForm.js`


You need to **reset** the form only   in block `then` in order not to do that immediately after `submit` because when there is an error, the modal will still be opened and the user can try to send the data again to the server without having to type the data into the inputs once more.